### DISCUSSION TITLE
AttributeError: 'dict' object has no attribute 'stores_as'

### DISCUSSION THREAD

**Discussion by moghadas76** [pyg-team/pytorch_geometric#8423] [Q&A]
### 🐛 Describe the bug

AttributeError: 'dict' object has no attribute 'stores_as'

### Environment

* PyG version:
* PyTorch version:
* OS:
* Python version:
* CUDA/cuDNN version:
* How you installed PyTorch and PyG (`conda`, `pip`, source):
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by akihironitta** [comment]
Would you mind sharing your code and env details to reproduce the error?
